tests/benchdnn/common.cpp

```diff
@@ -328,7 +328,7 @@ bool skip_start(res_t *res, int idx) {
     return false;
 }
 
-#if defined(_WIN32) && !defined(__GNUC__)
+#if defined(_WIN32)
```

Comment:
Please refer [this PR](https://github.com/oneapi-src/oneDNN/pull/224), which added `__GNUC__` guard in the first place. Could you elaborate on removing it, please?

Reply:
This file from benchdnn was not compiling even in x64 with MINGW. In MSYS packages, the unit tests are not build by default. I enabled them in x64 to check how to run the unit tests (and to later compare them against the ARM64 unit tests) and found that this fix was required.

It's clear from the call to `GlobalMemoryStatusEx();` that this should be the code path for Windows regardless of the architecture or compiler.

Reply:
For what it's worth, it appears that @msakai appears to have removed one of these guards after this comment: https://github.com/oneapi-src/oneDNN/pull/224#discussion_r184476692 it could be that these should also have been removed at the same time.

Reply:
@jondea that's not the same file / place in https://github.com/oneapi-src/oneDNN/pull/224 . My intent here is not to remove `&& !defined(__GNUC__)` . It really is just this one place.

Reply:
Indeed, but I think @rsdubtso's argument may still apply here too

Reply:
His comment was about Jit and function prologue. :(

Reply:
Fair enough, my knowledge is limited here so I'm happy to defer.